src/utils/wallet.js

```diff
@@ -926,6 +986,42 @@ const wallet = {
     storage.setItem(storageKeys.hideZeroBalanceTokens, false);
   },
 
+  /**
+   * Defines if a token should always be shown, despite having zero balance and the "hide zero
+   * balance" setting being active.
+   * @param {string} tokenUid uid of the token to be updated
+   * @param {boolean} [newValue=false] If true, the token will always be shown
+   */
+  setTokenAlwaysShow(tokenUid, newValue = false) {
```

Comment:
Why do you have this default for `newValue`? I feel we should require that it's always passed. What do you think?

Reply:
It's a good idea. Refactored on [57a4aac](https://github.com/HathorNetwork/hathor-wallet/pull/310/commits/57a4aac47afa185884ae4f9a6f4df11e16498d85).